app/src/main/java/com/x8bit/bitwarden/ui/platform/theme/Type.kt

```diff
@@ -218,38 +228,54 @@ val nonMaterialTypography: NonMaterialTypography = NonMaterialTypography(
         ),
         platformStyle = PlatformTextStyle(includeFontPadding = false),
     ),
-    bodySmallProminent = TextStyle(
+    eyebrowMedium = TextStyle(
         fontSize = 12.sp,
-        lineHeight = 16.sp,
-        fontFamily = FontFamily(Font(R.font.roboto_regular)),
+        lineHeight = 18.sp,
+        fontFamily = FontFamily(Font(R.font.dm_sans_bold)),
         fontWeight = FontWeight.W700,
-        letterSpacing = 0.4.sp,
-        lineHeightStyle = LineHeightStyle(
-            alignment = LineHeightStyle.Alignment.Center,
-            trim = LineHeightStyle.Trim.None,
-        ),
-        platformStyle = PlatformTextStyle(includeFontPadding = false),
-    ),
-    labelMediumProminent = TextStyle(
-        fontSize = 12.sp,
-        lineHeight = 16.sp,
-        fontFamily = FontFamily(Font(R.font.roboto_regular)),
-        fontWeight = FontWeight.W600,
         letterSpacing = 0.5.sp,
         lineHeightStyle = LineHeightStyle(
             alignment = LineHeightStyle.Alignment.Center,
             trim = LineHeightStyle.Trim.None,
         ),
-        platformStyle = PlatformTextStyle(includeFontPadding = false),
```

Comment:
Can we make sure the eyebrow has the `platformStyle = PlatformTextStyle(includeFontPadding = false),` too